modules/spo-creating-profiles.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-creating-profiles_{context}"]
+= Creating {type} profiles
+
+Use the `{kind}` object to create profiles.
+
+ifdef::seccomp[]
+
+`{kind}` objects can restrict syscalls within a container, limiting the access of your application.
+
+.Procedure
+
+* The {type} profile will be saved in `/var/lib/kubelet/{type}/operator/<namespace>/<name>.json`.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1beta1
+kind: {kind}
+metadata:
+  namespace: my-namespace
+  name: profile1
+spec:
+  defaultAction: SCMP_ACT_LOG
+----
+
+An `init` container creates the root directory of the Security Profiles Operator to run the Operator without `root` group or user ID privileges. A symbolic link is created from the rootless profile storage `/var/lib/openshift-security-profiles` to the default `seccomp` root path inside of the kubelet root `/var/lib/kubelet/{type}/operator`.
+endif::[]
+ifdef::selinux[]
+The `{kind}` object has several features that allow for better security hardening and readability. The `{kind}` object:
+
+* Restricts the profiles to inherit from to the current namespace or a system-wide profile. Because there are typically many profiles installed on the system, but only a subset should be used by cluster workloads, the inheritable system profiles are listed in the `spod` instance in `spec.selinuxOptions.allowedSystemProfiles`.
+* Performs basic validation of the permissions, classes and labels.
+* Adds a new keyword `@self` that describes the process using the policy. This allows reusing a policy between workloads and namespaces easily, as the usage of the policy is based on the name and namespace.
+* Adds features for better security hardening and readability compared to writing a profile directly in the SELinux CIL language.
```

Comment:
Lines 45-50 seem like they could be pulled out for an "About SelinuxProfile objects"-style module to appear before this procedure module, though maybe that would be more complicated than is worth considering the ifdefs at play here.